src/ProjectManager.cpp

```diff
@@ -871,8 +874,6 @@ void ProjectManager::OpenFiles(AudacityProject *proj)
       if (ProjectFileManager::IsAlreadyOpen(fileName))
          continue; // Skip ones that are already open.
 
-      if (!SafeToOpenProject(proj))
-         proj = nullptr;
```

Comment:
What was this removed function supposed to be doing, and why isn't it needed now?

Reply:
The call was introduced in an earlier commit in this branch, as part of the step-wise code transformation I like to leave in commit history, for purposes of easier review.

Now this check does not happen any more at this level, but is lowered into ProjectChooser.

I will make that clearer in commit comments.